src/healthz.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import redis
+from urllib.parse import urlparse
+
+from fastapi import APIRouter, HTTPException
+from sqlalchemy import text
+
+from datastores.sql import database
+
+router = APIRouter()
+
+def _check_posgresql_connection():
```

Comment:
nit: add type hint for return value (same for method below)

Reply:
Done